src/core/js/modelEvent.js

```diff
@@ -0,0 +1,32 @@
+define(function() {
+
+    var ModelEvent = Backbone.Controller.extend({
+
+        type: null,
+        target: null,
+        value: null,
+        bubbles: true,
```

Comment:
Minor thing, but since this is a Boolean could we possibly rename this to something like `canBubble` for readability?  To me `bubbles` sounds like it _could_ be an array.

Reply:
isn’t it named that to match https://developer.mozilla.org/en-US/docs/Web/API/Event/bubbles ?